gmail/src/models/attachment.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Represent an attachment in an email.
+ */
+export class Attachment {
```

Comment:
Isn't this class a bit redundant with the existing "GmailAttachment" ?

It's basically the same except for a few renames and encoding.

Reply:
Ok, I'll remove this model